aws-cloudformation-schema/aws-lambda-function.json

```diff
@@ -24,6 +24,17 @@
   "additionalProperties" : false,
   "primaryIdentifier" : [ "/properties/FunctionName" ],
   "definitions" : {
+    "SnapStartConfig": {
```

Comment:
What did you base this addition on, when the CF schema isn't released yet?

Reply:
A bit of guessing work based on advanced documentation releases plus Josh tested it against the early access we had enabled on our account and it seemed to work fine.

Reply:
Thanks. Just realized I can't approve it anyway since it's a draft.